installation.rst

```diff
@@ -198,9 +198,16 @@ Installation on MS Windows
 
 We are recommending and supporting installation of QuTiP into a Conda environment. Other scientific Python implementations such as Python-xy may also work, but are not supported.  
 
-As of QuTiP 4.1, recommended installation on Windows requires Python 3.5+, as well as Visual Studio 2015.  With this configuration, one can install QuTiP using any of the above mentioned receipes. Visual Studio 2015 is not required for the install of the conda-forge package, but it is required at runtime for the string format time-dependence solvers. When installing Visual Studio 2015 be sure to select options for the C++ compiler.
+As of QuTiP 4.1, recommended installation on Windows requires Python 3.5+, as well as MS Visual C++ Build Tools. MS Visual C++ Build Tools are strictly required for the install of the conda-forge package, but MS Visual Studio 2015 (or more recent) is recommended because it is required at runtime for the string format time-dependence solvers. When installing MS Visual Studio be sure to select the following components: 
```

Comment:
```suggestion
As of QuTiP 4.1, recommended installation on Windows requires Python 3.5+, as well as MS Visual C++ Build Tools. MS Visual C++ Build Tools are strictly required for the install of the conda-forge package, but [MS Visual Studio](https://visualstudio.microsoft.com/de/downloads/) 2015 (or more recent) is recommended because it is required at runtime for the string format time-dependence solvers. When installing MS Visual Studio be sure to select the following components: 
```